app/views/middleware_server/_deploy.html.haml

```diff
@@ -50,6 +50,17 @@
                                                      "placeholder" => _("No file chosen"),
                                                      "ng-model"    => "deployAddModel.runtimeName",
                                                      "ng-disabled" => "!deployAddModel.filePath"}
+          .form-group
+            %label.col-md-4.control-label{:for => "force_deployment_cb"}
+              = _("Overwrite (if exists)")
+            .col-md-8
+              %input#force_deployment_cb{:type             => "checkbox",
+                                         "bs-switch"       => "",
+                                         "switch-on-text"  => "Yes",
```

Comment:
Likely. Maybe this should go in a subsequent commit/PR as there are other occurrence in this same file needing it ?